frontend/packages/local-storage-operator-plugin/src/components/disks-list/disks-list-page.tsx

```diff
@@ -179,15 +179,15 @@ export const NodesDisksListPage: React.FC<NodesDisksListPageProps> = ({
       hideLabelFilter
       textFilter="node-disk-name"
       rowFilters={diskFilters}
-      flatten={(resource: FirehoseResult<LocalVolumeDiscoveryResultKind>) =>
-        resource[propName]?.data[0]?.status?.discoveredDevices ?? []
-      }
+      flatten={(
+        resource: FirehoseResourcesResult<{ [key: string]: LocalVolumeDiscoveryResultKind }>,
+      ) => resource[propName]?.data[0]?.status?.discoveredDevices ?? []}
       ListComponent={ListComponent ?? DisksList}
       resources={[
         {
           kind: referenceForModel(LocalVolumeDiscoveryResult),
           prop: propName,
-          selector: { [LABEL_SELECTOR]: nodeName },
+          selector: { matchLabels: { [LABEL_SELECTOR]: nodeName } },
```

Comment:
this looks like a bug as previous `selector` value is not valid

Reply:
> this looks like a bug as previous `selector` value is not valid

cc @afreen23

Reply:
I believe both are correct - one is equality based labelSelector ({ [LABEL_SELECTOR]: nodeName }) and  other is set based labelSelector which allows to create a complex set of selection AFAIK. 
Also, this currently works . You can try out discovering nodes after installing LSO operator.
So once you install LSO => visit Compute -> Nodes -> select a worker node -> disk -> run discovery (in few seconds you will see disks  populated under disks tab of that particular chosen node.)

Reply:
@afreen23 @andrewballantyne @spadgett found it https://github.com/openshift/console/blob/master/frontend/public/module/k8s/selector.ts#L40 - the original requests are the old format which still works.